expcore/Mini_games.lua

```diff
@@ -872,7 +873,7 @@ function Mini_games.print_results(results, options)
     local names = options.names or {}
     local limit = options.limit or 5
     for i, result in ipairs(results) do
-        if result.place < limit then
+        if result.place and result.place < limit then
             local place, score = Nth(result.place), result.score
```

Comment:
Think we will have an issue here if score is nil, could you add a check for this.
One option would be to have the default be an empty string which is tested for with a new condition on line 880.

Reply:
Having a placement without a score sounds odd, not sure what corner case could possibly make that, but I'll add a check for it.